globus_cli/config.py

```diff
@@ -86,12 +100,30 @@ def get_output_format():
     return lookup_option(OUTPUT_FORMAT_OPTNAME)
 
 
-def get_auth_refresh_token():
-    return lookup_option(AUTH_RT_OPTNAME)
+def get_auth_tokens():
+    return {
+        'refresh_token': lookup_option(AUTH_RT_OPTNAME),
+        'access_token': lookup_option(AUTH_AT_OPTNAME),
+        'access_token_expires': int(lookup_option(AUTH_AT_EXPIRES_OPTNAME))
```

Comment:
I meant using the helpers inside `lookup_option`, but yeah, since it's just two spots where we need it I'll handle it separately for now and we can always revisit if it pops up again.